components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIProviderImpl.java

```diff
@@ -138,19 +139,48 @@ public void updateBusinessOwner(BusinessOwner businessOwner) throws AppManagemen
      * @throws AppManagementException
      */
     @Override
-
     public List<BusinessOwner> getBusinessOwners() throws AppManagementException {
         return appMDAO.getBusinessOwners();
     }
 
     /**
+     * Get business owners.
+     * @param businessOwnerId Business owner Id.
+     * @return
+     * @throws AppManagementException
+     */
+    @Override
+    public BusinessOwner getBusinessOwner(int businessOwnerId) throws AppManagementException {
+        return appMDAO.getBusinessOwner(businessOwnerId);
+    }
+
+    /**
+     * Search business owners with pagination.
+     * @param startIndex
+     * @param pageSize
+     * @param searchValue
+     * @return
+     * @throws AppManagementException
+     */
+    @Override
+    public  List<BusinessOwner> searchBusinessOwners(int startIndex, int pageSize, String searchValue) throws
+                                                                                          AppManagementException {
+        return appMDAO.searchBusinessOwners(startIndex, pageSize, searchValue);
+    }
+
+    @Override
+    public  int getBusinessOwnersCount() throws AppManagementException {
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5